templates/themes/app/content/link.html.twig

```diff
@@ -0,0 +1,3 @@
+{% if location is defined %}
+    <a href="{{ ibexa_path(location) ~ suffix|default('') }}"{% if css_class is not empty %} class="{{ css_class }}"{% endif %}{% if new_tab is defined and new_tab %} target="_blank"{% endif %}>{{ label|default(content.name) }}</a>
```

Comment:
`suffix`, `label` and `new_tab` are already resolved in the `content_fields.html.twig` from where this view is rendered, so there is no need to default them or check if they are defined here.

Reply:
fixed in dd89f0183228c49c901709114784b63cf38e1f95